charts/gitops-runtime/Chart.yaml

```diff
@@ -15,10 +15,7 @@ annotations:
   artifacthub.io/alternativeName: "codefresh-gitops-runtime"
   artifacthub.io/changes: |
     - kind: changed
-      description: Update argo-cd chart to 5.53.12-4-cap-2.9-2024.2.22-25d9fca69
-      links:
-        - name: revert support application version custom path based on the Codefresh ApplicationConfiguration
-          url: https://github.com/codefresh-io/argo-cd/releases/tag/v2.9-2024.2.22-25d9fca69
+      description: Update app-proxy to 1.2667.0.
```

Comment:
please add 2 words on what was the change

Reply:
added